load.js

```diff
@@ -0,0 +1,55 @@
+var read = require('fs-readdir-recursive')
```

Comment:
Can we try to use [`globby`](https://github.com/sindresorhus/globby) instead? I believe we would do: (untested)

```js
var COMMON_PROTO_DIRS = [
  // This list of directories is defined here:
  // https://github.com/googleapis/googleapis/blob/master/gapic/packaging/common_protos.yaml
  'api',
  path.join('logging', 'type'),
  'longrunning',
  'protobuf', // This is an additional path that the common protos depend on.
  'rpc',
  'type'
].map(dir => path.join('google', dir, '**/*.proto'))

globby(COMMON_PROTO_DIRS).then(dirs => /* should already be flattened */)
```

Reply:
Ohhh nice. Done!